owncloudApp/src/main/java/com/owncloud/android/presentation/adapters/filelist/FileListAdapter.kt

```diff
@@ -115,5 +116,11 @@ class FileListAdapter(
         }
     }
 
-    inner class ViewHolder(val binding: ItemFileListBinding) : RecyclerView.ViewHolder(binding.root)
+    inner class ViewHolder(val binding: ItemFileListBinding) : RecyclerView.ViewHolder(binding.root) {
+        init {
+            itemView.setOnClickListener {
+                onItemClick?.invoke(files[adapterPosition])
```

Comment:
The `onItemClick()` should be with the rest of `bindings` in `onBindViewHolder()` method